### ISSUE TITLE
Ship experience is broken

### ISSUE THREAD

**Issue by dahaic** [ospaceteam/outerspace#56]
The way experience is calculated now is broken, as it is too valuable to micromanage.
- Bombing helps fighters get more experienced
- Pilot academies can teach heavy bomber crew lvl2, but these ships can be upgraded to fighter and get level 4 easily

Proposed solution:
Experience being points on defined scale, regardless of ship equipment, distributing based on individual efforts
- some points for dealt damage
- some points for received damage (can have a ceiling so you won't )

**Comment by temuchin-** [comment]
bombing helps fighter get xp faster, (I am aware of this feature).
Pilot academies can make bombers lvl2 , then upgrade to lvl4 (not able to do this, have been trying such a trick for 5+ years, game always scraps & caps xp to same level of new design)

**Comment by dahaic** [comment]
Good catch @temuchin-, it has been fixed years ago. I just haven't noticed.
